### ISSUE TITLE
[Suggestion] Option to enable / disable Vsync

### ISSUE THREAD

**Issue by RedNicStone** [DouglasDwyer/octo-release#1]
Hey first issue (sorry if this one is rather lame)!

I've been experimenting with the online demo, but unfortunately I was not able to archive a good frame time.
The frame time hovers between 16 and 18 ms and I am convinced that this is due to 60Hz Vsync (eg. 16.6ms).
My hardware is more than capable of running the demo at higher frame rates, and I would love to do so for the sake of testing the engines performance.
It would be awesome if either a switch for Vsync could be added or the frame timer adjusted to only account for the time spent drawing (using a fence or similar).

Thanks for providing this great demo!

**Comment by DouglasDwyer** [comment]
Hey, thanks for checking out the demo! I really appreciate it. Anyway, you surmise correctly that your framerate is being limited by Vsync. Unfortunately, this is not something that can be controlled on the web at the application level. The web browser implements Vsync, or not, and it's up to the user to control it.

You can check out [this article for how to disable Vsync in modern web browsers](https://cesium.com/blog/2014/12/01/webgl-profiling-tips/). With this, you should be able to attain higher performance! I probably should put a link to that article in some sort of FAQ. Let me know if you have any other questions, and feel free to reopen if the issue persists.

**Comment by DouglasDwyer** [comment]
Regarding your suggestion, it might be interesting in the future to try and add a more accurate frame timer that isn't dependent upon the time between command invocations. I would have to do a bit more reading about whether this is possible, but it looks like [sync objects are a part of the WebGL API](https://developer.mozilla.org/en-US/docs/Web/API/WebGLSync), so this could be possible.

**Comment by RedNicStone** [comment]
Hey thanks for your quick reply. Unfortunately I wasn't able to respond so soon.
The article you linked to was helpful, however I was only able to disable VSync on firefox.
I tried disabling "Sync to VBlank" (the only option regarding sync) in the driver as well as the `--disable-gpu-vsync`, which both did not help. After some research I found that chromes angular renderer would "software sync" to VBlank no matter what launch options are provided.
I'm glad it works on Firefox, and I'm AMAZED by the performance. It's hovering around 4-5ms with FXAA and the maximum render distance. Great work!
